common/desktop-exports

```diff
@@ -281,15 +281,19 @@ function make_user_fontconfig {
       echo "  <dir>${data_dirs_array[$i]}/fonts</dir>"
     fi
   done
+  # fix font render for modified fonts, that discussed on Snapcraft Forum:
+  # https://forum.snapcraft.io/t/snap-package-cannot-read-fonts-conf/16657
+  echo '  <include ignore_missing="yes">/etc/fonts/conf.d</include>'
```

Comment:
Does this rule expect to use the `fonts/conf.d` from the base snaps, or are you expecting it to read from the host?

If you are expecting it to read from the host then it won't work as expected.

Reply:
Ignore this, I just double checked - the directory is correctly accessible inside snaps. 🤦🏻‍♀️

Reply:
so it not need any edit?

Reply:
yup, the PR is good to go now :-)